Sources/HeapModule/Heap+UnsafeHandle.swift

```diff
@@ -81,12 +81,12 @@ extension Heap._UnsafeHandle {
 
   @inlinable @inline(__always)
   internal func minValue(_ a: _HeapNode, _ b: _HeapNode) -> _HeapNode {
-    self[a] < self[b] ? a : b
+    self[a] <= self[b] ? a : b
```

Comment:
I think it would make sense to add a comment explaining that the result emulates the stdlib. The implementation should also match precisely what the stdlib does, to emphasize this even further.

```suggestion
    // The expression used here matches the implementation of the 
    // standard `Swift.min(_:_:)` function. This attempts to 
    // preserve any pre-existing order in case `T` has identity.
    // `(min(x, y), max(x, y))` should return `(x, y)` in case `x == y`.
    self[b] < self[a] ? b : a
```